target/linux/realtek/image/rtl838x.mk

```diff
@@ -43,6 +43,15 @@ define Device/d-link_dgs-1210-20
 endef
 TARGET_DEVICES += d-link_dgs-1210-20
 
+define Device/d-link_dgs-1210-28mp-f
```

Comment:
"28mp-f" sorts after "28", so this block should ideally come right after the DGS-1210-28 block.